otsdaq-mu2e-tracker/HelloWorld/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cet_make_library(LIBRARY_NAME HelloWorld
+SOURCE
+	HelloWorld.cc
```

Comment:
I'm not entirely sure about committing this here. I believe that a template supervisor exists in otsdaq_demo, if not, that may be a more appropriate place for this.